superbench/benchmarks/reducer.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for result reducer."""
+
+from typing import Dict, Callable
+from statistics import mean
+
+from superbench.benchmarks.context import Enum
+
+
+class ReduceType(Enum):
+    """The Enum class representing different reducer."""
+    AVG = 'avg'
+    MAX = 'max'
+    MIN = 'min'
+    SUM = 'sum'
+
+
+class Reducer:
+    """Reducer class to maintain all reduce functions."""
+    functions: Dict[ReduceType, Callable] = dict()
```

Comment:
why not write the content here directly?

Reply:
Want to add reduce functions in a unified way by calling `add_reduce_func()`:

```
@Reducer.add_reduce_func(ReduceType.PERCENT99)
def percent_99(items):
    xxx
```